machines/aws/machine-set.yaml

```diff
@@ -28,6 +28,19 @@ spec:
           kind: AWSMachineProviderConfig
           ami:
             id: {{.AWS.Image}}
```

Comment:
What happens if you pass both here? Which does it choose? This seems like it should be optional.